qemu/tests/cfg/tpm_with_bitlocker.cfg

```diff
@@ -0,0 +1,22 @@
+- tpm_with_bitlocker:
+    only Windows
+    only q35
+    only ovmf
+    virt_test_type = qemu
+    type = tpm_with_bitlocker
+    required_qemu= [4.2.0,)
+    start_vm = yes
+    kill_vm = yes
+    clone_master = yes
+    master_images_clone = image1
+    remove_image_image1 = yes
+    tpms = tpm0
+    tpm_type_tpm0 = emulator
+    tpm_model_tpm0 = tpm-crb
+    tpm_version_tpm0 = 2.0
+    Win2019, Win2008, Win2016, Win2012:
+        cmd_install_bitlocker = 'powershell -command "Install-WindowsFeature -Name BitLocker"'
+    cmd_bdehdcfg = 'bdehdcfg -target c: shrink -newdriveletter s: -size 300 -quiet'
```

Comment:
Please help update -size 300 to -size 606. Thanks.

Reply:
Got it , have updated.